crypto/src/ed25519.rs

```diff
@@ -105,6 +105,12 @@ impl Display for Ed25519PublicKey {
     }
 }
 
+impl Debug for Ed25519PublicKey {
```

Comment:
I wouldn't change that in this PR (better keep it focused on ZeroCrypto). We can post it in another PR (even if tiny)

Reply:
Oops